src/index.js

```diff
@@ -6,7 +6,6 @@ import NavbarBrand from './NavbarBrand';
 import NavbarToggler from './NavbarToggler';
 import Nav from './Nav';
 import NavItem from './NavItem';
-import NavDropdown from './NavDropdown';
```

Comment:
You still need to import this and export it so that the deprecated component can still be used (and give people the warning).